lib/rubocop/cop/rspec/spec_file_path_format.rb

```diff
@@ -57,8 +63,67 @@ def on_top_level_example_group(node)
           end
         end
 
+        # For testing and debugging
+        def self.reset_activesupport_cache!
+          ActiveSupportInflector.reset_cache!
+        end
+
         private
 
+        # Inflector module that uses ActiveSupport for advanced inflection rules
+        module ActiveSupportInflector
+          def self.call(string)
+            ActiveSupport::Inflector.underscore(string)
+          end
+
+          def self.available?(cop_config)
```

Comment:
I am not sure if this method should memoize without considering `cop_config`.

Reply:
I think I need the config, but I will see what I can do

Reply:
I mean even if I separate the availability from the config check, I still need the config to look up the `inflector_path` and only in the case where it is `available?` or something like it.

So config is somewhat coupled to availability

Reply:
Best I can think of would be something like this, but that seems worse to me:

```ruby
        def active_support_available?
          return false unless cop_config.fetch(
            'UseActiveSupportInflections',
            false
          )

          ActiveSupportInflector.available?(cop_config)
        end

        def inflector
          @inflector ||= if active_support_available?
                           ActiveSupportInflector
                         else
                           DefaultInflector
                         end
        end
```        

Let me know whart you're aiming for here